Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.1.0"
 [deps]
 ArchGDAL = "c9ce4bd3-c3d5-55b8-8973-c0e20141b8c3"
 DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
+GeoInterface = "cf35fbd7-0cd7-5166-be24-54bfbe79505f"
```

Comment:
For now let's make this a test dependency only. You can simply move this line to an `[extras]` section, and then list GeoInterface in the `[targets]` section. See this example: https://github.com/JuliaEarth/Variography.jl/blob/f7fd09aa0a705701bf6eb93c52294cc0a13a99ad/Project.toml#L32-L48

Reply:
You will then need to load GeoInterface explicitly in runtests.jl

Reply:
I forgot to remove this dependency. We don't need it anymore.